### PR TITLE
Api page v2

### PR DESCRIPTION
**Pull request by AndrewKhukhrianskyi**
- Adds new tests
- Code refactoring
- Logout files deleting

### CODE REVIEW

File: spec/api/api_put_spec.rb [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+					api_cl.user_login(body[:username], body[:password])
+					response = api_cl.get_user(body[:username])
+					expect(response.status).to eq(200)
+					api_cl.user_logout()
+				end}
+				api_cl.delete_user(body[:username])	
+				
+				
+				# Negative Functional testing
+				it 'verifies that created user can be edited by using mixed data' do
```
Comment (AndrewKhukhrianskyi) [inline spec/api/api_put_spec.rb:59]: Bugs are detected here (From negative tests)

File: spec/api/api_put_spec.rb [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+					body[key] = value
+					api_cl.update_user(body[key], body)
+					api_cl.user_login(body[:username], body[:password])
+					response = api_cl.get_user(body[:username])
+					expect(response.status).to eq(200)
+					api_cl.user_logout()
+				end}
+				api_cl.delete_user(body[:username])	
+				
+
```
Comment (AndrewKhukhrianskyi) [inline spec/api/api_put_spec.rb:53]: Bugs are detected in the Negative PUT tests

File: spec/api/api_put_spec.rb [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+								  "email": 'kingofthenorth@gmail.com',
+								  "password": SecureRandom.hex,
+								  "phone": '0000000',
+								  "userStatus": 0}
+					valid_body.each_pair{|key, value| body[key] = value
+					api_cl.update_user(body[key], body)}
+					
+					api_cl.user_login(body[:username], body[:password])
+					response = api_cl.get_user(body[:username])
+					expect(response.status).to eq(200)
```
Comment (AndriiMish) [inline spec/api/api_put_spec.rb]: expect user body eq to valid_body check missing

File: spec/api/api_put_spec.rb [hunk 4]
```diff
@@ -0,0 +1,106 @@
+require_relative '../spec_helper'
+require_relative 'api_client.rb'
+require 'securerandom'
+
+RSpec.describe 'PUT method checking to the any data in the body' do
+	api_cl = ApiClient.new
+	body = api_cl.generate_random_body
+	
+	
+	api_cl.create_user(body)
```
Comment (AndriiMish) [inline spec/api/api_put_spec.rb:9]: this should be at lest in before block

File: spec/api/api_put_spec.rb [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+							  "username": 'user_name2',
+							  "firstName": 'Jon',
+							  "lastName": 'Snow',
+							  "email": 'nightswatch@gmail.com',
+							  "password": SecureRandom.hex,
+							  "phone": '11111111',
+							  "userStatus": 0}
+							  
+				api_cl.create_user(body)
+				valid_body.each_pair{|key, value|
```
Comment (AndriiMish) [inline spec/api/api_put_spec.rb:43]: missing 'do' keyword...does it pass?

**Pull request closed without merging**